source/val/validate_function.cpp

```diff
@@ -302,6 +304,7 @@ spv_result_t ValidateFunctionCall(ValidationState_t& _,
           case spv::StorageClass::Private:
           case spv::StorageClass::Workgroup:
           case spv::StorageClass::AtomicCounter:
+          case spv::StorageClass::NodePayloadAMDX:
```

Comment:
The spec seems to be missing this validation relaxation. Presuming this is intended, I'd suggest updating the extension spec.

Reply:
For now, I reverted this change.